vendor/helm.sh/helm/v3/pkg/action/action.go

```diff
@@ -84,6 +84,9 @@ type Configuration struct {
 	// KubeClient is a Kubernetes API client.
```

Comment:
Why is this vendored separately? Won't it get removed if I run `go mod vendor`? I'd expect such change to also have a change in go.mod and go.sum files.

Reply:
Right, I think this was a wrong rebase. Also, I'm seeing some problems with vendoring, I need to rework the vendoring. Stay tuned :)